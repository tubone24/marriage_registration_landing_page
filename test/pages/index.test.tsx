import React from 'react'
import { render } from 'test/testUtils'
import HomePage from 'src/pages'
import { RecoilRoot } from 'recoil'

global.fetch = require('node-fetch');

test('index snapshot', () => {
  const { container } = render(
    <RecoilRoot>
      <HomePage />
    </RecoilRoot>
    )
  expect(container).toMatchSnapshot()
})
