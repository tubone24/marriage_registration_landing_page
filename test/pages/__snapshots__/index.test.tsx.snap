// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index snapshot 1`] = `
<div>
  <div
    class="chakra-container css-fnlf1d"
  >
    <div
      class="chakra-stack css-1az190c"
    >
      <h2
        class="chakra-heading css-1xb57ov"
      >
        This is 
        <br />
        <span
          class="chakra-text css-9826r4"
        >
          Next.js
        </span>
        +
        <span
          class="chakra-text css-1xt4lt6"
        >
          TypeScript
        </span>
        +
        <br />
        <span
          class="chakra-text css-1oa66e7"
        >
          Chakra
        </span>
        +
        <span
          class="chakra-text css-9pvq78"
        >
          Jest
        </span>
        +
        <br />
        <span
          class="chakra-text css-1xslp7y"
        >
          StoryBook
        </span>
        +
        <span
          class="chakra-text css-3peusj"
        >
          Recoil
        </span>
        <br />
        Template
      </h2>
      <p
        class="chakra-text css-q9k0mw"
      >
        This template implements Next.js with TypeScript, uses ChakraUI for CSS, uses Jest for testing, StoryBook for cataloging components, and Recoil for state management by 
        <b>
          tubone24
        </b>
        .
      </p>
      <div
        class="chakra-stack css-grmpig"
      >
        <a
          href="https://github.com/tubone24/my-nextjs-template/fork"
        >
          <button
            class="chakra-button css-28gqvb"
            type="button"
          >
            Get Started
          </button>
        </a>
        <a
          href="https://github.com/tubone24/my-nextjs-template"
        >
          <button
            class="chakra-button css-1nbbexz"
            type="button"
          >
            Learn more
          </button>
        </a>
        <div
          class="css-0"
        >
          <svg
            class="chakra-icon chakra-icon css-1g3nvcs"
            focusable="false"
            viewBox="0 0 72 24"
          >
            <path
              clip-rule="evenodd"
              d="M0.600904 7.08166C0.764293 6.8879 1.01492 6.79004 1.26654 6.82177C2.83216 7.01918 5.20326 7.24581 7.54543 7.23964C9.92491 7.23338 12.1351 6.98464 13.4704 6.32142C13.84 6.13785 14.2885 6.28805 14.4722 6.65692C14.6559 7.02578 14.5052 7.47362 14.1356 7.6572C12.4625 8.48822 9.94063 8.72541 7.54852 8.7317C5.67514 8.73663 3.79547 8.5985 2.29921 8.44247C2.80955 9.59638 3.50943 10.6396 4.24665 11.7384C4.39435 11.9585 4.54354 12.1809 4.69301 12.4068C5.79543 14.0733 6.88128 15.8995 7.1179 18.2636C7.15893 18.6735 6.85928 19.0393 6.4486 19.0805C6.03792 19.1217 5.67174 18.8227 5.6307 18.4128C5.43271 16.4346 4.52957 14.868 3.4457 13.2296C3.3058 13.0181 3.16221 12.8046 3.01684 12.5885C2.05899 11.1646 1.02372 9.62564 0.457909 7.78069C0.383671 7.53862 0.437515 7.27541 0.600904 7.08166ZM5.52039 10.2248C5.77662 9.90161 6.24663 9.84687 6.57018 10.1025C16.4834 17.9344 29.9158 22.4064 42.0781 21.4773C54.1988 20.5514 65.0339 14.2748 69.9746 0.584299C70.1145 0.196597 70.5427 -0.0046455 70.931 0.134813C71.3193 0.274276 71.5206 0.70162 71.3807 1.08932C66.2105 15.4159 54.8056 22.0014 42.1913 22.965C29.6185 23.9254 15.8207 19.3142 5.64226 11.2727C5.31871 11.0171 5.26415 10.5479 5.52039 10.2248Z"
              fill="currentColor"
              fill-rule="evenodd"
            />
          </svg>
          <p
            class="chakra-text css-vxa5eg"
          >
            Use Template!!
          </p>
        </div>
      </div>
    </div>
  </div>
  <div
    class="css-h94677"
  >
    <div
      class="chakra-container chakra-stack css-d6kbj6"
    >
      <h2
        class="chakra-heading css-1n7pa12"
      >
        Feature
      </h2>
      <p
        class="chakra-text css-1gsj802"
      >
        This template contains a lot of modern features.
      </p>
    </div>
    <div
      class="chakra-container css-1780vv2"
    >
      <div
        class="css-1bma46p"
      >
        <div
          class="chakra-stack css-xiug77"
        >
          <div
            class="css-oict79"
          >
            <svg
              class="chakra-icon chakra-icon css-ikjmp3"
              focusable="false"
              viewBox="0 0 14 14"
            >
              <g
                fill="currentColor"
              >
                <polygon
                  points="5.5 11.9993304 14 3.49933039 12.5 2 5.5 8.99933039 1.5 4.9968652 0 6.49933039"
                />
              </g>
            </svg>
          </div>
          <div
            class="chakra-stack css-xerlbm"
          >
            <p
              class="chakra-text css-35ezg3"
            >
              Next.js
            </p>
            <p
              class="chakra-text css-jneyc"
            >
              We use Next.js, so we are familiar with SPA, SSG, and API development.
            </p>
          </div>
        </div>
        <div
          class="chakra-stack css-xiug77"
        >
          <div
            class="css-oict79"
          >
            <svg
              class="chakra-icon chakra-icon css-ikjmp3"
              focusable="false"
              viewBox="0 0 14 14"
            >
              <g
                fill="currentColor"
              >
                <polygon
                  points="5.5 11.9993304 14 3.49933039 12.5 2 5.5 8.99933039 1.5 4.9968652 0 6.49933039"
                />
              </g>
            </svg>
          </div>
          <div
            class="chakra-stack css-xerlbm"
          >
            <p
              class="chakra-text css-35ezg3"
            >
              Chakra UI
            </p>
            <p
              class="chakra-text css-jneyc"
            >
              By using Chakra UI, you can easily achieve CSS in JS while achieving Utility First.
            </p>
          </div>
        </div>
        <div
          class="chakra-stack css-xiug77"
        >
          <div
            class="css-oict79"
          >
            <svg
              class="chakra-icon chakra-icon css-ikjmp3"
              focusable="false"
              viewBox="0 0 14 14"
            >
              <g
                fill="currentColor"
              >
                <polygon
                  points="5.5 11.9993304 14 3.49933039 12.5 2 5.5 8.99933039 1.5 4.9968652 0 6.49933039"
                />
              </g>
            </svg>
          </div>
          <div
            class="chakra-stack css-xerlbm"
          >
            <p
              class="chakra-text css-35ezg3"
            >
              TypeScript
            </p>
            <p
              class="chakra-text css-jneyc"
            >
              I've heard from my grandpa that if you don't use TypeScript, you'll get very angry.
            </p>
          </div>
        </div>
        <div
          class="chakra-stack css-xiug77"
        >
          <div
            class="css-oict79"
          >
            <svg
              class="chakra-icon chakra-icon css-ikjmp3"
              focusable="false"
              viewBox="0 0 14 14"
            >
              <g
                fill="currentColor"
              >
                <polygon
                  points="5.5 11.9993304 14 3.49933039 12.5 2 5.5 8.99933039 1.5 4.9968652 0 6.49933039"
                />
              </g>
            </svg>
          </div>
          <div
            class="chakra-stack css-xerlbm"
          >
            <p
              class="chakra-text css-35ezg3"
            >
              Jest(UI SnapShot)
            </p>
            <p
              class="chakra-text css-jneyc"
            >
              By using Jest (UI SnapShot), you can run regression tests safely and easily.
            </p>
          </div>
        </div>
        <div
          class="chakra-stack css-xiug77"
        >
          <div
            class="css-oict79"
          >
            <svg
              class="chakra-icon chakra-icon css-ikjmp3"
              focusable="false"
              viewBox="0 0 14 14"
            >
              <g
                fill="currentColor"
              >
                <polygon
                  points="5.5 11.9993304 14 3.49933039 12.5 2 5.5 8.99933039 1.5 4.9968652 0 6.49933039"
                />
              </g>
            </svg>
          </div>
          <div
            class="chakra-stack css-xerlbm"
          >
            <p
              class="chakra-text css-35ezg3"
            >
              StoryBook
            </p>
            <p
              class="chakra-text css-jneyc"
            >
              Storybook is an open source tool for developing UI components and pages in isolation.
            </p>
          </div>
        </div>
        <div
          class="chakra-stack css-xiug77"
        >
          <div
            class="css-oict79"
          >
            <svg
              class="chakra-icon chakra-icon css-ikjmp3"
              focusable="false"
              viewBox="0 0 14 14"
            >
              <g
                fill="currentColor"
              >
                <polygon
                  points="5.5 11.9993304 14 3.49933039 12.5 2 5.5 8.99933039 1.5 4.9968652 0 6.49933039"
                />
              </g>
            </svg>
          </div>
          <div
            class="chakra-stack css-xerlbm"
          >
            <p
              class="chakra-text css-35ezg3"
            >
              Recoil
            </p>
            <p
              class="chakra-text css-jneyc"
            >
              By using Recoil, it is now possible to manage the state of functional components in a relatively simple way.
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="css-w8mx6m"
  >
    <div
      class="chakra-container css-o44ltb"
      direction="column"
    >
      <h2
        class="chakra-heading css-takvb8"
      >
        Your name (Recoil Test)
      </h2>
      <form
        class="chakra-stack css-1e2li35"
      >
        <div
          class="chakra-form-control css-1kxonj9"
          role="group"
        >
          <input
            aria-label="Your Name"
            aria-required="true"
            class="chakra-input css-1nvrhak"
            id="name"
            placeholder="Your Name"
            required=""
            type="text"
            value="tubone24"
          />
        </div>
        <div
          class="chakra-form-control css-555mxg"
          role="group"
        >
          <button
            class="chakra-button css-59wu70"
            type="submit"
          >
            Submit
          </button>
        </div>
      </form>
      <p
        class="chakra-text css-10pmlql"
      >
        You won't receive any spam! ✌️
      </p>
    </div>
  </div>
  <div
    class="chakra-stack css-tf1ayk"
  >
    <p
      class="chakra-text css-r65o2g"
    >
      I've had more and more opportunities to create everything with Next.js, but it's getting to be a hassle to modify the project from create-next-app to my liking every time. So I decided to create a template to speed up the development process.
    </p>
    <div
      class="css-xi606m"
    >
      <span
        alt="tubone24"
        class="chakra-avatar css-1j8889u"
      >
        <svg
          aria-label=" avatar"
          class="chakra-avatar__svg css-16ite8i"
          role="img"
          viewBox="0 0 128 128"
        >
          <path
            d="M103,102.1388 C93.094,111.92 79.3504,118 64.1638,118 C48.8056,118 34.9294,111.768 25,101.7892 L25,95.2 C25,86.8096 31.981,80 40.6,80 L87.4,80 C96.019,80 103,86.8096 103,95.2 L103,102.1388 Z"
            fill="currentColor"
          />
          <path
            d="M63.9961647,24 C51.2938136,24 41,34.2938136 41,46.9961647 C41,59.7061864 51.2938136,70 63.9961647,70 C76.6985159,70 87,59.7061864 87,46.9961647 C87,34.2938136 76.6985159,24 63.9961647,24"
            fill="currentColor"
          />
        </svg>
      </span>
      <p
        class="chakra-text css-35ezg3"
      >
        tubone24
      </p>
      <p
        class="chakra-text css-zgbaaa"
      >
        Rubbish Engineer
      </p>
    </div>
  </div>
  <div
    class="css-1gx57aq"
  >
    <div
      class="chakra-stack chakra-container css-faj5gh"
    >
      <p
        class="chakra-text css-0"
      >
        © 
        
         tubone24. All rights reserved
      </p>
      <div
        class="chakra-stack css-1amf41o"
      >
        <a
          class="css-hyb7bs"
          href="https://twitter.com/meitante1conan"
        >
          <span
            class="css-f8n5zr"
          >
            Twitter
          </span>
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 512 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"
            />
          </svg>
        </a>
        <a
          class="css-hyb7bs"
          href="https://github.com/tubone24"
        >
          <span
            class="css-f8n5zr"
          >
            GitHub
          </span>
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 496 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
            />
          </svg>
        </a>
      </div>
    </div>
  </div>
</div>
`;
